recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch

```diff
@@ -0,0 +1,78 @@
+From b736741e8b5db849b9c698a8bbdd2f3a3549c714 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Fri, 24 Sep 2021 11:50:29 +0800
+Subject: [PATCH 9/9] u-boot: dts : remove usb3.0 related dts configurations
+
+This is a known issue that usb3.0 can not work under u-boot.
+The usb3.0 port can only work at usb2.0 mode. So remove the usb3.0
+related dts configurations,and disable the usb3.0 serdes signal.
+
+These modification should be removed when the usb3.0 is ready under u-boot
+
+Signed-off-by: chao zeng <chao.zeng@siemens.com>
+---
+ arch/arm/dts/k3-am65-iot2050-u-boot-pg2.dtsi  | 26 +++++++++++++++++++
+ arch/arm/dts/k3-am6528-iot2050-basic-pg2.dts  |  1 +
+ .../dts/k3-am6548-iot2050-advanced-pg2.dts    |  1 +
+ 3 files changed, 28 insertions(+)
+ create mode 100644 arch/arm/dts/k3-am65-iot2050-u-boot-pg2.dtsi
+
+diff --git a/arch/arm/dts/k3-am65-iot2050-u-boot-pg2.dtsi b/arch/arm/dts/k3-am65-iot2050-u-boot-pg2.dtsi
+new file mode 100644
+index 0000000000..76d95b38c4
+--- /dev/null
++++ b/arch/arm/dts/k3-am65-iot2050-u-boot-pg2.dtsi
+@@ -0,0 +1,26 @@
++// SPDX-License-Identifier: GPL-2.0
++/*
++ * Copyright (c) Siemens AG, 2018-2021
++ *
++ * Authors:
++ *   Chao Zeng <chao.zeng@siemens.com>
++ *
++ * U-Boot bits of the IOT2050 Advanced PG2 variants
++ */
++
++&serdes0 {
++	status = "disabled";
++};
++
++&dwc3_0 {
++	assigned-clock-parents = <&k3_clks 151 4>,	/* set REF_CLK to 20MHz i.e. PER0_PLL/48 */
++					 <&k3_clks 151 9>;	/* set PIPE3_TXB_CLK to CLK_12M_RC/256 (for HS only) */
++	/delete-property/ phys;
++	/delete-property/ phy-names;
++};
++
++&usb0_phy {
++	maximum-speed = "high-speed";
++	/delete-property/ snps,dis-u1-entry-quirk;
++	/delete-property/ snps,dis-u2-entry-quirk;
++};
+diff --git a/arch/arm/dts/k3-am6528-iot2050-basic-pg2.dts b/arch/arm/dts/k3-am6528-iot2050-basic-pg2.dts
+index c62549a4b4..95b21ce959 100644
+--- a/arch/arm/dts/k3-am6528-iot2050-basic-pg2.dts
++++ b/arch/arm/dts/k3-am6528-iot2050-basic-pg2.dts
+@@ -17,6 +17,7 @@
+ 
+ #include "k3-am6528-iot2050-basic-common.dtsi"
+ #include "k3-am65-iot2050-common-pg2.dtsi"
++#include "k3-am65-iot2050-u-boot-pg2.dtsi"
+ 
+ / {
+ 	compatible = "siemens,iot2050-basic-pg2", "ti,am654";
+diff --git a/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts b/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts
+index f00dc86d01..1420ba1041 100644
+--- a/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts
++++ b/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts
+@@ -17,6 +17,7 @@
+ 
+ #include "k3-am6548-iot2050-advanced-common.dtsi"
+ #include "k3-am65-iot2050-common-pg2.dtsi"
++#include "k3-am65-iot2050-u-boot-pg2.dtsi"
```

Comment:
Why not add this to k3-am65-iot2050-common-pg2.dtsi? Then you only need to touch one file.

Reply:
the usb dts configuration for pg2 exists in the k3-am65-iot2050-common-pg2.dtsi.
it is simple to remove or modify these node. but you mentioned before in order to keep sync with kernel side, it should not modify this file.  So add a new file for the specific change for the pg2 board.

Reply:
OK, ordering matters here: We need to include this after the common pg2 bits.